app/src/pages/inside/common/dashboardPageHeader/addDashboardButton/addDashboardButton.jsx

```diff
@@ -33,8 +34,26 @@ const messages = defineMessages({
     id: 'DashboardForm.addModalTitle',
     defaultMessage: 'Add New Dashboard',
   },
+  buttonTooltip: {
+    id: 'DashboardHeaderButton.buttonTooltip',
+    defaultMessage: `The limit of 300 dashboards has been reached. To create a new one you need to delete at least one created previously`,
+  },
 });
 
+const TooltipComponent = ({ formatMessage }) => <p>{formatMessage(messages.buttonTooltip)}</p>;
+
```

Comment:
Extra empty line. When using prop types with functional components, please keep them right after the component definition without extra lines.